go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/roc-project/roc-go
+
```

Comment:
Shouldn't we also add GO111MODULE=on to `env` section in travis?

Reply:
alright

Reply:
I'm wondering how did the ci pass :thinking:

Reply:
Hm, mys suggestion was not quite correct. By just adding a line to `env` we tell travis to build all combinations of these three exports:

```
  - GOTESTFLAGS=""
  - GOTESTFLAGS="-race"
  - GO111MODULE=on
```

![image](https://user-images.githubusercontent.com/8436629/84873264-a72e6600-b08b-11ea-9e00-cd24d693afb8.png)

We should use `global`: https://docs.travis-ci.com/user/environment-variables/#global-variables

Reply:
fixed

Reply:
The build matrix is broken now for some reason.

Was: https://travis-ci.org/github/roc-project/roc-go/builds/699219954

![image](https://user-images.githubusercontent.com/8436629/84944515-8c410d80-b0ee-11ea-9c45-03dd45b6abdd.png)

Now: https://travis-ci.org/github/roc-project/roc-go/builds/699428591

![image](https://user-images.githubusercontent.com/8436629/84944534-9400b200-b0ee-11ea-8b97-dd6251451ec2.png)

Reply:
Maybe try to remove `- ` from `- global:` and `- jobs`.

Reply:
https://config.travis-ci.com/explore

Reply:
I'll take a look into it

Reply:
yes, don't know what I was thinking